### DISCUSSION TITLE
Add an Instafade option/toggle

### DISCUSSION THREAD

**Discussion by RyanStephenX** [ppy/osu#18789] [Ideas] [answered]
Instafade is short for instant fading skin where there is no hitcircle animations when you click the circles. Instafade is made through the merging of hitcircle, hitcircleoverlay, to the default numbers. Making the hitcircle and hitcircleoverlay element a transparent png.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/5913. Please search before reporting.
